<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyUserDialog</class>
 <widget class="QDialog" name="ModifyUserDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>280</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  border: 1px solid gray;
  margin-top: 10px;
  padding: 5px;
}
QGroupBox::title {
  top: -10px;
  left: 10px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>确认密码：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLineEdit" name="userNameEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="userPasswordEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmUserPasswordEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="confirmButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  border: 1px solid gray;
  margin-top: 10px;
  padding: 5px;
}
QGroupBox::title {
  top: -10px;
  left: 10px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>用户级别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>终端类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>终端ID：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QComboBox" name="userLevelBox">
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="terminalTypeBox">
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="terminalIDEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  border: 1px solid gray;
  padding: 5px;
}
QGroupBox::title {
  top: -10px;
  left: 10px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>服务截至时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="serviceDeadlineEdit">
      <property name="displayFormat">
       <string>yyyy年MM月dd hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
